test/integration/env/test_env.py

```diff
@@ -0,0 +1,33 @@
+import os
+import re
+import subprocess
+
+from utils import config_name
+
+
+def test_env_output(tt_cmd, tmpdir):
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
+        f.write('env:\n  bin_dir:\n  inc_dir:\n')
+    binDir = str(tmpdir + "/bin")
+    tarantoolDir = "TARANTOOL_DIR=" + str(tmpdir + "/include/include/tarantool")
```

Comment:
`/include/include`?

Reply:
Yes, include files in tt are stored in "IncludeDir + include/program".